portal/models/user.py

```diff
@@ -1171,7 +1171,8 @@ def fuzzy_match(self, first_name, last_name, birthdate):
         # birthdate is trickier - raw delta doesn't make sense.  treat
         # it like a string, mismatch always results in a 0 score
         dob = self.birthdate or datetime.utcnow()
-        if (dob.strftime('%d%m%Y') != birthdate.strftime('%d%m%Y')):
+        if (birthdate.year < 1900 or
+            dob.strftime('%d%m%Y') != birthdate.strftime('%d%m%Y')):
```

Comment:
I don't believe we should ever have a member with a birthday < 1900.  The problem is the string given to strftime will raise an exception if the year within is < 1900.